src/middlewared/middlewared/plugins/service.py

```diff
@@ -896,23 +896,20 @@ def _started_notify(self, verb, what):
         await self._service("netatalk", "reload", **kwargs)
 
     async def _restart_cifs(self, **kwargs):
-        await self._service("ix-pre-samba", "start", quiet=True, **kwargs)
+        await self._system("/usr/local/bin/python /usr/local/libexec/nas/generate_smb4_conf.py")
```

Comment:
A python script using shell to call another python script really bugs me : P.
I know you're keeping it because of HA but why exactly it needs to stay that way?
Couldnt it be moved to etc plugin as well but with `py` type?

Reply:
Moved those to etc plugin with py type.